### PR TITLE
Use standard comment container on subject page

### LINKED ISSUE #1655: The role labels, user avatars and comment details (date, time) are not showing on the subject page
The role labels, user avatars and comment details (date, time) are not showing on the subject page. Tested on Gorongosa and Whales, using Firefox on Ubuntu, and then on Chrome on Yoga tablet. Both the same results.

Like here: https://www.zooniverse.org/projects/tedcheese/whales-as-individuals/talk/subjects/481016

screenshot_2015-09-27_18-46-21
screenshot_2015-09-27_18-47-15

### LINKED ISSUE #1700: Autoscaling down images that are too big that are in a comment posted on the Talk Subject page
Example [here] (https://www.zooniverse.org/projects/mschwamb/planet-four-terrains/talk/subjects/490275/)

screen shot 2015-10-05 at 9 42 38 pm

### PR DESCRIPTION
**Pull request by aweiksnar**
- Closes #1700
- Closes #1655

### CODE REVIEW

File: app/subjects/index.cjsx [hunk 1]
```diff
@@ -28,28 +30,38 @@ module?.exports = React.createClass
...
-  comment: (data, i) ->
-    <CommentLink key={data.id} comment={data}>
-      <div className="talk-module">
-        <strong>{data.user_display_name}</strong>
-        <br />
-        <Markdown project={@props.project}>{data.body}</Markdown>
-      </div>
-    </CommentLink>
+  comment: (comment, i) ->
+    <Comment data={comment} locked={true} linked={true} />
```
Comment (chrissnyder) [inline app/subjects/index.cjsx]: Needs a key prop.

Comment (chrissnyder) [inline app/subjects/index.cjsx]: Are `locked` and `linked` always true on the subject's page?

Comment (aweiksnar) [inline app/subjects/index.cjsx]: Just added the :key: prop.
`locked` hides the reply/edit/report buttons, and `linked` adds in that extra link,  which I think always make sense on the subject page in it's current state 

That could change with #1271

Comment (chrissnyder) [inline app/subjects/index.cjsx]: :+1:

**Comment by chrissnyder** [comment]
:shipit:

**Pull request merged**